internal/resources/azure/data_factory_integration_runtime_azure_ssis.go

```diff
@@ -0,0 +1,89 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// DataFactoryIntegrationRuntimeAzureSSIS struct represents Data Factory's
+// Azure-SSIS runtime.
+//
+// Resource information: https://azure.microsoft.com/en-us/services/data-factory/
+// Pricing information: https://azure.microsoft.com/en-us/pricing/details/data-factory/ssis/
+type DataFactoryIntegrationRuntimeAzureSSIS struct {
+	Address string
+	Region  string
+
+	Instances       int64
+	InstanceType    string
+	Enterprise      bool
+	LicenseIncluded bool
+}
+
+// DataFactoryIntegrationRuntimeAzureSSISUsageSchema defines a list which represents the usage schema of DataFactoryIntegrationRuntimeAzureSSIS.
+var DataFactoryIntegrationRuntimeAzureSSISUsageSchema = []*schema.UsageItem{}
+
+// PopulateUsage parses the u schema.UsageData into the DataFactoryIntegrationRuntimeAzureSSIS.
+// It uses the `infracost_usage` struct tags to populate data into the DataFactoryIntegrationRuntimeAzureSSIS.
+func (r *DataFactoryIntegrationRuntimeAzureSSIS) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+// BuildResource builds a schema.Resource from a valid DataFactoryIntegrationRuntimeAzureSSIS struct.
+// This method is called after the resource is initialised by an IaC provider.
+// See providers folder for more information.
+func (r *DataFactoryIntegrationRuntimeAzureSSIS) BuildResource() *schema.Resource {
+	costComponents := []*schema.CostComponent{
+		r.computeCostComponent(),
+	}
+
+	return &schema.Resource{
+		Name:           r.Address,
+		UsageSchema:    DataFactoryIntegrationRuntimeAzureSSISUsageSchema,
+		CostComponents: costComponents,
+	}
+}
+
+// computeCostComponent returns a cost component for cluster configuration.
+func (r *DataFactoryIntegrationRuntimeAzureSSIS) computeCostComponent() *schema.CostComponent {
+	productType := "Standard"
+	if r.Enterprise {
+		productType = "Enterprise"
+	}
+
+	license := "License Included"
+	licenseTitle := " with license"
+	if !r.LicenseIncluded {
+		license = "AHB"
+		licenseTitle = ""
+	}
+
+	instanceTitle := "instance"
+	if r.Instances > 1 {
+		instanceTitle = "instances"
+	}
+
+	return &schema.CostComponent{
+		Name:           fmt.Sprintf("Compute (%s%s, %d %s)", productType, licenseTitle, r.Instances, instanceTitle),
```

Comment:
One thing we've done in other cases is use the `UnitMultiplier` to take the time component out of the the unit (e.g. for S3 storage we use "GB" for the unit instead of "GB-month".  In this case would mean something like:
```
Unit: "instance"
UnitMultiplier: schema.HourToMonthUnitMultiplier,
```
but I don't know if that would be better.  It probably depends on how the prices are discussed in the azure docs/cost calculator.

Reply:
This is a cost component for AWS' RedShift cluster:
```go
			Name:           fmt.Sprintf("Cluster usage (%s, %s)", "on-demand", r.NodeType),
			Unit:           "hours",
			UnitMultiplier: decimal.NewFromInt(1),
			HourlyQuantity: decimalPtr(decimal.NewFromInt(numberOfNodes)),
```

I think I should just follow this example to stay consistent. We can iterate on making the multipliers more visible later.